spec.html

```diff
@@ -21438,6 +21743,31 @@ <h1>Runtime Semantics: ClassFieldDefinitionEvaluation</h1>
       </emu-note>
     </emu-clause>
 
+    <emu-clause id="sec-runtime-semantics-classstaticblockdefinitionevaluation" type="sdo" aoid="ClassStaticBlockDefinitionEvaluation">
+      <h1>Runtime Semantics: ClassStaticBlockDefinitionEvaluation</h1>
+      <p>With parameter _homeObject_.</p>
+      <emu-grammar>ClassStaticBlock: `static` `{` ClassStaticBlockBody `}`</emu-grammar>
+      <emu-alg>
+        1. Let _lex_ be the running execution context's LexicalEnvironment.
+        1. Let _privateScope_ be the running execution context's PrivateEnvironment.
+        1. Let _sourceText_ be the empty sequence of Unicode code points.
+        1. Let _bodyFunction_ be OrdinaryFunctionCreate(%Function.prototype%, _sourceText_, &laquo; &raquo;, |ClassStaticBlockBody|, ~non-lexical-this~, _lex_, _privateScope_).
```

Comment:
OrdinaryFunctionCreate's 3rd parameter is a Parse Node, not a List.
```suggestion
        1. Let _formalParameters_ be an instance of the production <emu-grammar>FormalParameters : [empty]</emu-grammar>.
        1. Let _bodyFunction_ be OrdinaryFunctionCreate(%Function.prototype%, _sourceText_, _formalParameters_, |ClassStaticBlockBody|, ~non-lexical-this~, _lex_, _privateScope_).
```